CMSgov/mint-app
# [EASI-3324] - All help articles - HKC

# EASI-3324

## Changes and Description

- Added view for all help aricles
- Added link on HKC home page to view all
- Added pagination support/prop to `SolutionHelpCardGroup`
- Updated `<RelatedHelpArticles />` component to direct to all articles
- Added unit test/snapshot



## How to test this change

- Navigate to HKC and click the link to `View all help articles`
- Verify the articles render with their tags
- To view pagination alter this line to `3` instead of `9`
  - https://github.com/CMSgov/mint-app/blob/29bdc94c440b8cf124b1cb0163f3dd6a7f6c89a4/src/views/HelpAndKnowledge/Articles/_components/HelpCardGroup/index.tsx#L35

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
